scalingo/resource_scalingo_alert.go

```diff
@@ -210,7 +218,7 @@ func resourceAlertsUpdate(ctx context.Context, d *schema.ResourceData, meta inte
 			"send_when_below":         alertUpdate.SendWhenBelow,
 			"duration_before_trigger": alertUpdate.DurationBeforeTrigger.String(),
 			"remind_every":            alertUpdate.RemindEvery,
-			"notifiers":               alertUpdate.Notifiers,
+			"notifiers":               alertUpdateParams.Notifiers,
```

Comment:
Why are you using `alertUpdateParams` here?
Is the `client.AlertUpdate` not returns the correct notifier?

Reply:
The GET by id and the PATCH routes aren't returning the notifiers, so I have to save them from the input parameters